README.md

```diff
@@ -82,6 +82,10 @@ node /path/to/repository/packages/ckeditor5-package-generator <packageName> [--d
 
 Available scripts and their modifiers are described in the [`README.md` file of the `ckeditor5-package-generator` package](/packages/ckeditor5-package-generator).
 
+#### Package metadata
+
+The `ckeditor5-metadata.json` file contains data of the package that allows for an automated detection of plugins and processing them by external scripts. Information about how this file should be maintained is available in the [official guide](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/framework/guides/contributing/package-metadata.html). Keep in mind that this file has no effect on how the plugin work.
```

Comment:
Also I think it would be better to link to the official version of the docs rather than to the nightly.